### PR TITLE
(microsite) Updating the roadmap on backstage.io

### PR DESCRIPTION
**Pull request by leemills83**
Updating the Backstage roadmap with current and upcoming work through H1 2021.

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)

### CODE REVIEW

File: docs/overview/roadmap.md [hunk 1]
```diff
@@ -47,55 +47,34 @@ guidelines to get started.
...
+  Creating an easy, standardized way for developers to build plugins will encourage contributions and lead to a richer ecosystem for everyone.
 
-- **[Initial GraphQL API](https://github.com/backstage/backstage/milestone/13)** -
+- **[GraphQL API](https://github.com/backstage/backstage/milestone/13)** -
   A GraphQL API will open up the rich metadata provided by Backstage in a single
   query. Plugins can easily query this API as well as extend the model where
   needed.
 
+- **Inter Plugin Communication** -  **[Under consideration]** Establish more clearly defined patterns for plugins to communicate.
+Improved Access Control -   [Under consideration] Provide finer grained access controls and management making it possible for you to better control the platforms user experience.
```
Comment (adamdmharvey) [inline docs/overview/roadmap.md:75]: duplicate bullet, hidden under the previous one?

**Review by freben — APPROVED** [review]
@leemills83 You'll have to run prettier on this.

**Comment by OrkoHunter** [comment]
And for the CI / Vale error, add `composable` to `.github/styles/vocab.txt`. Vale did not recognise that as a word.

File: docs/overview/roadmap.md (hunk 1 above)
Comment (leemills83) [reply docs/overview/roadmap.md]: Good catch, thanks!

**Comment by benjdlambert** [comment]
Now you'll need to do `yarn prettier docs/overview/roadmap.md` and commit the changes :)

**Comment by dtuite** [comment]
> @leemills83 You'll have to run prettier on this.

Someone needs to build a bot which posts this message on every PR 😂

**Comment by dtuite** [comment]
@leemills83 This is not a review, but a random thought as I read through the changes.

I wonder does it make sense to have a KPI for the Backstage OSS project? Typically a roadmap would be in service to some KPI which you are trying to improve. Perhaps something like unique contributors per week or number of companies in ADOPTERS.md. [This Tweet thread](https://twitter.com/_changxu/status/1337410288811794433?s=20) has some other ideas about KPIs for open source projects.

It might be overkill at this point but just a thought.

**Comment by benjdlambert** [comment]
@leemills83 I went ahead and fixed this up for you - feel free to merge when you're ready.

**Comment by leemills83** [comment]
> @leemills83 I went ahead and fixed this up for you - feel free to merge when you're ready.

Thanks dude! I just got it working in my dev environment, so ready for next time ;)

**Comment by leemills83** [comment]
@AlleyWei can you check out the update before I merge

**Comment by leemills83** [comment]
> @leemills83 This is not a review, but a random thought as I read through the changes.
> 
> I wonder does it make sense to have a KPI for the Backstage OSS project? Typically a roadmap would be in service to some KPI which you are trying to improve. Perhaps something like unique contributors per week or number of companies in ADOPTERS.md. [This Tweet thread](https://twitter.com/_changxu/status/1337410288811794433?s=20) has some other ideas about KPIs for open source projects.
> 
> It might be overkill at this point but just a thought.

Good thought! let us do some thinking to see if theres something we should add at this stage 😄

**Comment by AlleyWei** [comment]
@leemills83 from a content standpoint, this looks good! @fengypants and I are reviewing the original doc, though, for editing checks. You can either incorporate those then merge, or merge this, and I'll create a PR for those updates. LMK how you want to approach.

**Pull request merged**